app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -29,5 +31,19 @@ def destroy
       form.destroy
       render json: form
     end
+
+    private
+
+    def ensure_uuid
+      # There have been several errors where `@current_user.uuid` is being coerced to `nil`
+      # by activerecord. This checks the the `uuid` against the same regex and logs an error
+      # if we see a 'malformed' id.
+      uuid_format = ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Uuid::ACCEPTABLE_UUID
```

Comment:
I just looked this up and it takes the form:
```
%r{\A\{?([a-fA-F0-9]{4}-?){8}\}?\z}x
```
My regex is weak - does the `{4}-?` part mean it is supposed to contain 4 dashes?  The UUID in question never contains dashes.